manticore/ethereum/manticore.py

```diff
@@ -1455,6 +1410,13 @@ def unregister_detector(self, d):
     def workspace(self):
         return self._workspace._store.uri
 
+    @property
+    def global_findings(self):
+        try:
+            return self._global_findings
+        except AttributeError:
+            return set()
```

Comment:
Should you also set `self._global_findings` here?

Reply:
Detectors and findings are an issue.  This should be available only after finalization. I'm considering adding a manticore "State" for this. 
 
Before every time a Detector finds a tentative issue it used to add it in the global list immediately. So the findings were reported even if the state in which they have been found later REVERTED of did not reach a final condition.